service/src/main/java/org/mskcc/cmo/metadb/service/util/RequestDataFactory.java

```diff
@@ -86,8 +86,11 @@ private static RequestMetadata extractRequestMetadataFromJson(String requestMeta
         if (requestMetadataMap.containsKey("samples")) {
             requestMetadataMap.remove("samples");
         }
+        Object requestId = requestMetadataMap.containsKey("requestId")
```

Comment:
I'm thinking that we use the same logic to resolve the genePanel above instead of the objectutils